### PR TITLE
feat(FEC-14691):adding an option to display entry title on upper bar

### PR DESCRIPTION
**Pull request by ravitshalem**
### Description of the Changes

Add configuration to display entry name on the upper bar

### CODE REVIEW

File: src/components/title/title.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  }
+
+  const textRef = useRef<HTMLSpanElement | null>(null);
+  const comparisonTextRef = useRef<HTMLSpanElement | null>(null);
+  const [showTooltip, setShowTooltip] = useState(false);
+  const [isFinalized, setIsFinalized] = useState(false);
+  const [width, setWidth] = useState(0);
+
+  let title = '';
+  try {
```
Comment (SivanA-Kaltura) [inline src/components/title/title.tsx]: you don't need a try catch if you already test that the value is not empty
